microceph/ceph/services.go

```diff
@@ -138,6 +138,64 @@ func getMons() (common.Set, error) {
 	return retval, nil
 }
 
+func getActiveMons() ([]string, error) {
```

Comment:
The file services.go was originally written to host methods for all services however, we added files for individual services later (monitor.go for mon, metadata.go for mds, and manager.go for mgr). It would be nicer to have these methods in the respective files.

Reply:
oops, I missed this comment. Updated now